src/mdbExtensionController.ts

```diff
@@ -168,6 +168,21 @@ export default class MDBExtensionController implements vscode.Disposable {
     this.showOverviewPageIfRecentlyInstalled();
     void this.showSurveyForEstablishedUsers();
     void this.showCopilotIntroductionForEstablishedUsers();
+
+    const copilot = vscode.extensions.getExtension('GitHub.copilot');
+    void vscode.commands.executeCommand(
+      'setContext',
+      'mdb.isCopilotActive',
+      copilot?.isActive
+    );
+
+    vscode.extensions.onDidChange(() => {
+      void vscode.commands.executeCommand(
+        'setContext',
+        'mdb.isCopilotActive',
+        copilot?.isActive
```

Comment:
Could it be that this value doesn't actually update in this case? Would it not use old set variable and we might need to freshly run `vscode.extensions.getExtension('GitHub.copilot')` here ?

Reply:
I have manually tested and this updated the value and shows/hides the icon accordingly.

Reply:
Played with this more and removed the `vscode.extensions.onDidChange` listener. See for more details the vscode ticket: https://github.com/microsoft/vscode/issues/234426